conan/tools/cmake/cmakedeps/templates/macros.py

```diff
@@ -0,0 +1,88 @@
+import textwrap
+
+from conan.tools.cmake.cmakedeps.templates import CMakeDepsFileTemplate
+
+"""
+
+cmakedeps_macros.cmake
+
+"""
+
+
+class MacrosTemplate(CMakeDepsFileTemplate):
+    """cmakedeps_macros.cmake"""
+
+    def __init__(self):
+        super(MacrosTemplate, self).__init__(req=None, configuration=None)
+
+    @property
+    def filename(self):
+        return "cmakedeps_macros.cmake"
+
+    @property
+    def context(self):
+        return {}
+
+    @property
+    def template(self):
+        return textwrap.dedent("""
+        function(conan_message MESSAGE_OUTPUT)
+            if(NOT CONAN_CMAKE_SILENT_OUTPUT)
+                message(${ARGV${0}})
+            endif()
+        endfunction()
+
+       macro(conan_find_apple_frameworks FRAMEWORKS_FOUND FRAMEWORKS FRAMEWORKS_DIRS)
+           if(APPLE)
+               foreach(_FRAMEWORK ${FRAMEWORKS})
+                   message("BUSCANDO FRAMEWORK: ${_FRAMEWORK}")
```

Comment:
Better English message

Reply:
hahaha I forgot to remove the debug "pincho"